ansible/provision-private-proxied-cluster-airgap-rpm.yml

```diff
@@ -0,0 +1,57 @@
+### This is not a truely airgapped cluster!!! The intent here is specifically to test the install flow for an airgapped install
+
+---
+- name: Provision client
+  hosts: client
+  vars:
+    squid_acl_localnet:
+      - "10.0.1.0/24"
+  tasks:
+    - name: Install and configure proxy on client
+      ansible.builtin.include_role:
+        name: mrlesmithjr.squid
+
+- name: Bundle RPMs
+  hosts: client
+  vars:
+    basearch: "x86_64" # must match the output of `uname -m`
+    os_distribution_major_version: "34" # must match the output of `. /etc/os-release && echo ${VERSION_ID%%.*}`
+    os_distribution: "fedora" # must match the output of `. /etc/os-release && echo $ID`
+    redpanda_version: "23.1.13-1" # note that the -1 is the build number for this version. it is typically -1 but if the version is incremented it will be higher
+    rpm_or_deb: "rpm"
+    ansible_ssh_common_args: "-o ProxyCommand='ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /app/artifacts/testkey -W %h:%p -q {{ hostvars[groups['client'][0]].ansible_user }}@{{ hostvars[groups['client'][0]].inventory_hostname }}'"
+  tasks:
+    - name: Install and configure proxy on client
+      ansible.builtin.include_role:
+        name: redpanda.cluster.binary_bundler
+      run_once: true
+
+## clients[0] is obviously pretty ugly, need to come up with an az friendly way to handle this
+## BUT because it is in the playbook it is easy for the user to adjust to their liking at least
```

Comment:
Can you change the wording here, please? maybe add a TODO with the specific linked issue for future ref.